src/pages/saksbehandling/vedtak/klagevedtaksoppsummering/klagevedtaksoppsummering.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import OppsummeringAvKlage from '~components/oppsummeringAvKlage/OppsummeringAvKlage';
+import { Klage } from '~types/Klage';
+import { Vedtak } from '~types/Vedtak';
+
+interface Props {
+    vedtak: Vedtak;
+    klage: Klage;
+}
+const Klagevedtaksoppsummering = (props: Props) => {
+    return <OppsummeringAvKlage klage={props.klage} klagensVedtak={props.vedtak} />;
+};
```

Comment:
Ser ikke helt hvorfor dette trenger å være en eget komponent, og ikke bare kalle direkte på `OppsummeringAvKlage`?

Reply:
Mest for å ha samme struktur som de andra komponentene. Per nå så har jag ikke lagt til titler og ting på siden ännu men da blir det fort mer oversiktligt å ha det i en egen komponenten tenker jag, istedet for å edit:a det inline. 😊